data-structure.py

```diff
@@ -2,22 +2,9 @@ def read_file(text_file):
   if ".txt" in text_file:
     recipe_list = open(text_file)
     opened_recipe_list = (recipe_list.read())
-    return opened_recipe_list
+    print opened_recipe_list
   else:
     print ("Invalid Input: Not a .txt file")
 
+def data_structure(string_input):

```

Comment:
What is the need for an empty function here?

Reply:
Hey!  This is why I put the empty function.